contracts/OpenSTUtility.sol

```diff
@@ -69,12 +69,14 @@ contract OpenSTUtility is Hasher, OpsManaged, STPrimeConfig {
 
 
     /*
-  *  Constants
-  */
-    // ~2 weeks, assuming ~15s per block
-    uint256 public constant BLOCKS_TO_WAIT_LONG = 80667;
-    // ~1hour, assuming ~15s per block
-    uint256 public constant BLOCKS_TO_WAIT_SHORT = 240;
+     *  Constants
+     */
+    // ~2 weeks
+    uint256 private constant TIME_TO_WAIT_LONG = 1209600;
+    // ~5Days
+    uint256 private constant TIME_TO_WAIT_MEDIUM = 432000;
```

Comment:
the `medium wait` is not needed in OpenSTUtility and OpenSTValue, per our discussion?